source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,87 @@
+.. _php-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: setup, composer, installation, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure that you have the following
+      dependencies installed on your local machine:
+      
+      - `PHP <https://www.php.net/manual/en/install.php>`__
```

Comment:
Q: Which version of PHP/Composer are necessary?

Reply:
added!